### ISSUE TITLE
Release the trained models in PyTorch format too, not just ONNX

### ISSUE THREAD

**Issue by SupremeLobster** [Ant-Brain/EfficientWord-Net#51]
Would it be possible for you to release the trained ResNet-50_Arcloss model in PyTorch (.pt) format? It would be easier to perform operations like quantization.

Thanks

**Comment by TheSeriousProgrammer** [comment]
Hi thanks for the pr, its just the model weights and the dataset which where present in our academic VM got deleted, so that's kind of a bummer. But still I will attempt to caliibrate the onnx model with calibration like mentioned in the docs and will update you

**Comment by SupremeLobster** [comment]
> Hi thanks for the pr, its just the model weights and the dataset which where present in our academic VM got deleted, so that's kind of a bummer. But still I will attempt to caliibrate the onnx model with calibration like mentioned in the docs and will update you

Yeah not problem. I am running this system (with quantization) in production. It runs on an Angular frontend using onnxruntine. The model seems to run just fine even on mobile phones. I didnt need to retrain anything, just having good references is good enough (you need variety in the references tho). Thanks for your work!
This system I'm deploying is part of my Master's Thesis, and I am citing this work of yours

**Comment by TheSeriousProgrammer** [comment]
Thank you for the kind words, kindly share us your thesis upon completion, I'll happily add it in community projects 😄
